### DISCUSSION TITLE
Another noob question

### DISCUSSION THREAD

**Discussion by Xcoffeerr** [rusq/slackdump#139] [Q&A] [answered]
Mac laptop. Downloaded slackbot zip file, opened and it's now in my Downloads folder as a folder with this name: "slackdump-v2.2.0-alpha-darwin-amd64". I can open and run it: when it asks what I'd like to do I say "dump," and when asks me to list conversations I enter the channel ID from slack, and it says it's fetching, but can't find where it is dumping info. The last few lines of the Terminal code are:

logout
Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.

[Process completed]

I have gotten this to work before but am just not able to figure out what to do next. (I have read through all of the FAQs and Q&As and such already. I have searched all over my hard drive. I am fairly new at using github and slackdump.)

**Comment by rusq** [comment, marked as answer]
Hey @Xcoffeerr , if you're running the "Dump" mode, it should create a JSON file with data it exported.

To help me understand what's happening, could you do the following:
1. Navigate in Finder to where the slackdump-v2.2.0-alpha-darwin-amd64 folder is;
2. Right click on it, choose "Services->New Terminal At Folder" (the terminal window will open)
3. Type: the folloing command:
```
ls -l
```
4. Post the output here.

Then

1. Type the following in the SAME terminal:
```
./slackdump
```
2. Do what you did before
3. After it completes - 
```
ls -l
```
And copy and paste the full output of slackdump and `ls -l` command.

Let me know how it goes?

**Comment by Xcoffeerr** [comment]
Thanks for your response!

I don’t know what happened, but when I followed your steps just now and
went to copy the output I noticed that the .json file was saved to the
folder. It’s a mystery but it now is working, so thank you!
